src/index.html

```diff
@@ -5,8 +5,459 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page-content">
+    <header class="header">
+      <a href="#home">
+        <img
+         src="images/logo.png"
+         alt="logo Moyo"
+         class="logo"
+        >
+      </a>
```

Comment:
i think according to the rules BEM, it's better to do it like this

a
<
   class="header__logo"
>